app/main.py

```diff
@@ -1,3 +1,48 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[Distance, int]) -> Union[Distance, int]:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        self.km = self.km + other
+        return self
+
+    def __iadd__(self, other: Union[Distance, int]) -> Union[Distance, int]:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, other: int) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Distance) -> bool:
+        return self.km < other
```

Comment:
The `__lt__` method should handle both `Distance` objects and integers. You need to check if `other` is an instance of `Distance` and compare accordingly, or directly compare with an integer.